openshift-client-python/README.md

```diff
@@ -0,0 +1,64 @@
+# Python OpenShift Runner
+[OpenShift](http://www.openshift.com) is a Kubernetes distribution from Red Hat which provides `oc`, the [OpenShift CLI](https://docs.openshift.com/container-platform/4.1/cli_reference/getting-started-cli.html) that complements `kubectl` for simplifying deployment and configuration applications on OpenShift.
+The following task can be used to interact with the openshift cluster via `oc` commands in a python script.
+
+The set of possible scripts can be found in the docs [here](https://github.com/openshift/openshift-client-python/blob/master/README.md#usage). 
+
+## Pre-requisites
+
+1. Need to have an OpenShift cluster up and running.
+2. `config` file of the cluster needs to be mounted via the `ConfigMap` to the default location `~/.kube/` in order to obtain the cluster details.
+
+## Install the Task
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/openshift-client-python/openshift-client-python.yaml
+```
+
+## Parameters
+
+* **SCRIPT**: The `python` script which you want to run in the task. Default script would be :-
+```
+#!/usr/bin/python
+
+import openshift as oc
+print('OpenShift client version: {}'.format(oc.get_client_version()))
+print('OpenShift server version: {}'.format(oc.get_server_version()))
+```
+
+## Workspaces
+
+* **kubeconfig-mount**: The following workspace is used to mount the `config` file of the `kubeconfig` from the `ConfigMap`.
```

Comment:
Does the kubeconfig include auth information that should be kept secret?

Reply:
Replaced with `Secrets` instead of `ConfigMap`